docs/blog/2019-02-26-getting-started-with-gatsby-themes/index.md

```diff
@@ -173,18 +156,62 @@ Your workspace info should look similar to this:
 
 Lastly, you're going to want to add a _gatsby-config.js_ file to your _site_ directory.
 
-```javascript:title=gatsby-config.js
+```javascript:title=site/gatsby-config.js
 module.exports = {
-  __experimentalThemes: ["theme"],
+  __experimentalThemes: [`theme`],
+}
+```
+
+### Setting up Site `package.json
+
+You will need to add `gatsby` CLI scripts and specify your newly created `theme` as a dependency.
+
+```json:title=site/package.json
+{
+  "name": "site",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    // highlight-start
+    "develop": "gatsby develop",
+    "build": "gatsby build"
+    // highlight-end
+  },
+  "dependencies": {
+    "gatsby": "^2.3.34",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    // highlight-start
+    "theme": "*"
+    // highlight-end
```

Comment:
Does this work (e.g. in the actual blog post)? It's invalid JSON--but may work in the resulting built site?

Reply:
It does appear to work, I'm guessing the magic highlight commands are ignored during parsing.

![image](https://user-images.githubusercontent.com/1424573/57038163-805a6980-6c16-11e9-8d48-1369142f45ee.png)

Reply:
Works for me--I can't imagine anyone is looking at our blog posts from Github (whereas e.g. README or something else is more reasonable!)